### DISCUSSION TITLE
Is there a way to draw arrows ?

### DISCUSSION THREAD

**Discussion by hugo73cr** [sgratzl/chartjs-chart-graph#107] [Q&A] [answered]
Hello, I have a tree graph and I want to display arrow at the end of my lines. Is there a way to do this ?
Thanks !

**Comment by sgratzl** [comment, marked as answer]
see https://www.sgratzl.com/chartjs-chart-graph/examples/directed.html

**Reply by hugo73cr** [reply]
Oh thanks, I missed this part of the doc

**Reply by hugo73cr** [reply]
If anyone don't see the arrows after applied the directed: true, your nodes maybe hide them, you need to add arrowHeadOffset
